basic-search-java/src/main/java/com/mydomain/example/MetalSearcher.java

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.mydomain.example;
+
+import com.google.inject.Inject;
+import com.yahoo.prelude.query.IndexedItem;
+import com.yahoo.prelude.query.Item;
+import com.yahoo.prelude.query.OrItem;
+import com.yahoo.prelude.query.WordItem;
+import com.yahoo.search.Query;
+import com.yahoo.search.Result;
+import com.yahoo.search.Searcher;
+import com.yahoo.search.query.Model;
+import com.yahoo.search.query.QueryTree;
+import com.yahoo.search.searchchain.Execution;
+import com.yahoo.yolean.chain.After;
```

Comment:
Unused import ... but maybe you should use it.

Reply:
yes, used to - moved the after to services.xml but can put both places just for the example